### DISCUSSION TITLE
Playback of Previous Chat Speeches with Silero_TTS

### DISCUSSION THREAD

**Discussion by alireza20144** [oobabooga/text-generation-webui#2235] [Q&A]
Hey there!
I just wanted to share my excitement about the text-generation-webui. It's been an amazing tool that has helped me dive into the world of AI learning machines.
But here's the thing: I recently (now!) updated the webui and downloaded the snakers4_silero-models_master. Since then, I've been experiencing a little hiccup. Whenever I use the silero_tts feature during a chat, it starts playing all the previous chat speeches while generating the new speech. Has anyone else encountered this? I'd love to hear if you've found a solution or if you have any tips to make it solved.
Alireza

**Comment by bitjockeypro** [comment]
Yep, same here. Using silero_tts will playback all previously generated responses, instead of just the latest one.

**Comment by alireza20144** [comment]
The issue with the silero_tts feature in the text-generation web UI has been resolved. After updating and cleaning the caches, the playback of previous voice responds has stopped. Thanks to the developers and the community for their support. Happy exploring!
